src/quo/components/info/info_message.cljs

```diff
@@ -11,6 +11,7 @@
   (case k
     :success (colors/resolve-color :success theme)
     :error   (colors/resolve-color :danger theme)
+    :warning (colors/resolve-color :orange theme)
```

Comment:
Added the missing type in the component.

Reply:
bit of a nit - but in the foundations file there is color "warninig". it seems to be the same color but perhaps we should also add this as :warning in the map `colors-map` where success and danger are added. This way we can update the color in future much easier if the use cases changes. i.e right now we are coupling user orange with warning orange

Reply:
> bit of a nit - but in the foundations file there is color "warninig"

I couldn't able to find `warning` in `quo.foundations.colors` ns. Could you help me with which ns you find it? If not, I will add those colors in this PR.

> but perhaps we should also add this as :warning in the map colors-map where success and danger are added. This way we can update the color in future much easier if the use cases changes. i.e right now we are coupling user orange with warning orange

Yes, we should be using the warning color instead of the user color. I will bring this up with the design team as in the design system, this component uses the user color (orange).
Nice catch!

Reply:
Ah warning is not in the colors file, we should add it in this pr! 🙏